src/guides/cache.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: cache
+title: Cache
 author: l7ssha
 timestamp: 2021-09-21
-category: guides
+category: Guides
 ---
 
-Cache in nyxx is managed automatically with internal wrappers around existing Dart features. Nyxx features 
-[`Cache<T, S>`](https://nyxx.l7ssha.xyz/dartdocs/nyxx/nyxx/Cache-class.html) interface which is base class for all cache types used in library. 
-But like any other aspect of nyxx cache handling can be altered to fit your needs.
+Caching in Nyxx is managed automatically with internal wrappers around existing Dart features. Nyxx features a 
```

Comment:
also leave nyxx lower case, it looks better that way and its been 'branding' for a long time now